### PR TITLE
Upgrade notes

### PR DESCRIPTION
**Pull request by ultramegatom**

### CODE REVIEW

File: doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Upgrade Notes
 
+## 6.2.3
```
Comment (weisswurstkanone) [inline doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md]: next one will 6.3.0

Comment (weisswurstkanone) [reply doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md]: see https://github.com/pimcore/pimcore/milestones

Comment (ultramegatom) [reply doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md]: but its already present in 6.2.3

Comment (ultramegatom) [reply doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md]: @see https://github.com/pimcore/pimcore/issues/4740

Comment (weisswurstkanone) [reply doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md]: since 6.1.0 ?

Comment (ultramegatom) [reply doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md]: ohh yes sry missread that - moved it down to 6.1.0

**Comment by weisswurstkanone** [comment]
merci

**Pull request merged**